tf-modules/cumulus_ecs_service/main.tf

```diff
@@ -41,6 +41,7 @@ resource "aws_ecs_task_definition" "default" {
         options = {
           awslogs-group  = aws_cloudwatch_log_group.default.name
           awslogs-region = data.aws_region.current.name
+          awslogs-stream-prefix = var.name
```

Comment:
The log stream will be like:
HelloWorld/jl-rds-tf-HelloWorld/f241293d748e478ba899158a728c92b9
instead of:
a566bea365ba6833097acd564ad65c6f73737b71b58447d0eae773811226a3bf